play-2.4/swagger-play2/app/play/modules/swagger/SwaggerPlugin.scala

```diff
@@ -81,9 +82,9 @@ class SwaggerPluginImpl @Inject()(lifecycle: ApplicationLifecycle, router: Route
   }
 
   val routesFile = config.underlying.hasPath("play.http.router") match {
-    case false => "routes"
+    case false => "conf/routes"
     case true => config.getString("play.http.router") match {
-      case None => "routes"
+      case None => "conf/routes"
       case Some(value)=> value.replace(".Routes", ".routes")
```

Comment:
Ok, got it and changed it; only doubt was to leave the custom option without "/conf" to allow files in any path, so not necessarily only "conf", but I believe this is not really needed/wanted